pandas/core/computation/ops.py

```diff
@@ -541,7 +542,7 @@ def __unicode__(self):
 class FuncNode(object):
 
     def __init__(self, name):
-        if name not in _mathops:
+        if name not in _mathops or not hasattr(np, name) :
```

Comment:
why is this change needed?

Reply:
so older numpy does not support floor and ceil methods. They were introduced only in the 1.15. So, i felt it would be more appropriate to check if its supported instead of having a version check.